src/react-components/src/components/01-atoms/Forms/Checkbox.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * A Form Checkbox component that can be selected and deselected.
+ */
+
+import * as React from "react";
+import bem from "../../../utils/bem";
+import Label, { LabelOptions } from "./Label";
+
+export interface CheckboxOptions {
+  name?: string;
+
+  modifiers?: string[];
+  blockName?: string;
+  labelOptions?: LabelOptions;
+
+  isSelected?: boolean;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+interface CheckboxProps {
+  /** name of the checkbox */
+  name?: string;
+
+
+  /** Used for BEM css convention. */
+  blockName?: string;
+  /** Used for BEM css convention. */
+  modifiers?: string[];
+
+  /** Unique ID used by checkbox */
+  checkboxId: string;
+  /* The Label that the checkbox is using. */
+  labelOptions: LabelOptions;
+
+  /* The current selected state of the checkbox */
+  isSelected?: boolean;
+  /** The action to perform on the <input>'s onChange function  */
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
I think this code isn't working as intended—in Storybook, I've never able to apply the checked state to the checkbox